recipes/eleuther_eval.py

```diff
@@ -13,6 +12,12 @@
 import PIL
 
 import torch
+
+from lm_eval.evaluator import evaluate, get_task_list
+from lm_eval.models.hf_vlms import HFMultimodalLM
+from lm_eval.models.huggingface import HFLM
+from lm_eval.tasks import get_task_dict, TaskManager
+from lm_eval.utils import make_table
```

Comment:
Don't we still need to gate here? lm_eval is still not in our pyproject.toml

Reply:
Yeah I was going to default to just let the ModuleNotFoundError roll through since it seems explicit enough.